packages/driver/cypress/integration/e2e/multi-domain/multi_domain_validation_specs.ts

```diff
@@ -101,8 +101,28 @@ describe('multi-domain', () => {
       })
     })
 
+    it('succeeds on a url with path', () => {
+      cy.origin('http://www.foobar7.com/login', () => undefined)
+      cy.then(() => {
+        const expectedSrc = `http://foobar7.com/__cypress/multi-domain-iframes`
+        const iframe = window.top?.document.getElementById('Spec\ Bridge:\ http://foobar7.com') as HTMLIFrameElement
+
+        expect(iframe.src).to.equal(expectedSrc)
+      })
+    })
+
+    it('succeeds on a url with a hash', () => {
+      cy.origin('http://www.foobar8.com/#hash', () => undefined)
+      cy.then(() => {
+        const expectedSrc = `http://foobar8.com/__cypress/multi-domain-iframes`
+        const iframe = window.top?.document.getElementById('Spec\ Bridge:\ http://foobar8.com') as HTMLIFrameElement
+
+        expect(iframe.src).to.equal(expectedSrc)
+      })
+    })
+
```

Comment:
Should we have tests for hostname + path and hostname + hash? What about ones with path _and_ hash?